mne/viz/topomap.py

```diff
@@ -167,11 +167,10 @@ def _average_fnirs_overlaps(info, ch_type, sphere):
                 # first listed channel is the one to be replaced with merge
                 overlapping_set = [chs[i]['ch_name'] for i in
                                    np.where(overlapping_mask[chan_idx])[0]]
-                overlapping_set = np.insert(overlapping_set, 0,
-                                            (chs[chan_idx]['ch_name']))
+                overlapping_set.insert(0,chs[chan_idx]['ch_name'])
```

Comment:
style error:
```
./mne/viz/topomap.py:170:41: E231 missing whitespace after ','
```
```suggestion
                overlapping_set.insert(0, chs[chan_idx]['ch_name'])
```

Reply:
style error:
```
./mne/viz/topomap.py:170:41: E231 missing whitespace after ','
```

Reply:
dangit, the GitHub interface isn't letting me apply the suggested change. @JohnGriffiths can you see if you have the "commit suggestion" button available, and if not, make this change locally and push?

Reply:
@drammock - I pressed the button as requested for your change with the whitespace addition.